### ISSUE TITLE
Getting Started tutorial references renamed and no longer existing streams in v1

### ISSUE THREAD

**Issue by dvorapa** [bluefireteam/audioplayers#1200]
Getting Started tutorial references some streams, that are renamed or no longer exist in v1. I've caught at least these three:
```
  error • The getter 'onAudioPositionChanged' isn't defined for the type
         'AudioPlayer' • lib/main.dart:1004:17 • undefined_getter
  error • The getter 'onPlayerCompletion' isn't defined for the type
         'AudioPlayer' • lib/main.dart:1023:17 • undefined_getter
  error • The getter 'onPlayerError' isn't defined for the type 'AudioPlayer' •
         lib/main.dart:1033:17 • undefined_getter
```
The first two obviously have just been renamed. They should be renamed in the tutorial as well. But the third one seems no longer existing in the audioplayers package - at least I haven't found it yet.

**Code to Reproduce**

1. Open the Getting Started tutorial ( https://github.com/bluefireteam/audioplayers/blob/main/getting_started.md )
2. Add audioplayers as a dependency into some Flutter app
3. Copy and paste the code examples for the three streams mentioned above directly from Getting Started tutorial to your Flutter app
4. Run `flutter pub get && flutter analyze`

**Platforms**
* flutter version: 3.0.4
* audioplayers version: 1.0.1

**Comment by dvorapa** [comment]
The last one covered by #1179
The second one fixed in #1189

**Comment by Gustl22** [comment]
First one via: #1275
